vpype/io.py

```diff
@@ -336,6 +336,7 @@ def write_svg(
     layer_label_format: str = "%d",
     show_pen_up: bool = False,
     color_mode: str = "none",
+    no_basic_shapes: bool = False
```

Comment:
I dislike this name. Back then, that feature was `single_path`, which I'd use here.

Reply:
i renamed. i agree that your's is the better one